examples/object_detection_2d/object_detection_2d/seed_test_run.py

```diff
@@ -116,15 +115,13 @@ def setup_evaluator() -> ObjectDetectionEvaluator:
     return ObjectDetectionEvaluator(
         configurations=[
             ThresholdConfiguration(
-                threshold_strategy=ThresholdStrategy.FIXED_03,
+                threshold_strategy=0.3,
                 iou_threshold=0.3,
-                with_class_level_metrics=True,
                 min_confidence_score=0.2,
             ),
             ThresholdConfiguration(
-                threshold_strategy=ThresholdStrategy.F1_OPTIMAL,
+                threshold_strategy="F1-Optimal",
```

Comment:
I think `threshold_strategy: Union[Literal["F1-Optimal"], float]` is a good way for users to define a `number` or `"F1-Optimal"` for the strategy. It feels most intuitive that way for me, much better than `enums`

Reply:
I would avoid using a specific string as a parameter because it's something we would have to look it up every time to get the string match exactly. 

And it looks like if a slightly different version of "F1-Optimal" like "f1-optimal" or "f1" is used then it still accepts it but it will not work as expected.